contract_sale/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Contract from Sale',
+    'version': '10.0.1.0.0',
+    'category': 'Sales',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/contract',
+    'depends': [
+        'sale',
```

Comment:
Shouldn't this be enough with `sales_team`?

Reply:
Yes, it's enough